xbmc/guilib/GUIControl.cpp

```diff
@@ -185,7 +186,15 @@ void CGUIControl::DoRender()
       g_graphicsContext.SetStereoFactor(m_stereo);
 
     GUIPROFILER_RENDER_BEGIN(this);
+
+    if (m_hitColor != 0xFFFFFFFF)
```

Comment:
@FernetMenta would adding `CGUIInfoColor& operator!=(color_t color);` be enough?